### PR TITLE
[FE] Login - Logout - Register - I18N

### PR DESCRIPTION
**Pull request by DTHNhung**
## Related Tickets:

-   [#45911](https://edu-redmine.sun-asterisk.vn/projects/ecommerce-intern-hn/issues)

## WHAT this PR does?

- Login - Logout - Register - I18N

## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ok

**Comment by DTHNhung** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -13,5 +14,11 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
+Route::get('/', function () {
+    return view('welcome');
+});
```
Comment (huyenttm-0901) [inline routes/web.php]: ko viết logic trong route

File: routes/web.php [hunk 2]
```diff
@@ -13,5 +14,11 @@
...
 
 Route::get('/lang/{locale}', [LanguageController::class, 'switchLanguage'])->name('lang');
+Auth::routes();
+
+Route::get('/home', [\App\Http\Controllers\HomeController::class, 'index'])->name('home');
```
Comment (huyenttm-0901) [inline routes/web.php]: Use cái namespace này cho dễ nhìn đi em

**Comment by DTHNhung** [comment]
ready

File: webpack.mix.js [hunk 3]
```diff
@@ -6,12 +6,21 @@ const mix = require('laravel-mix');
...
-        //
+        require('postcss-import'),
+        require('tailwindcss'),
+        require('postcss-nested'),
+        require('autoprefixer'),
     ]);
+
+if (mix.inProduction()) {
+    mix
+        .version();
```
Comment (ghost) [inline webpack.mix.js]:
```suggestion
    mix.version();
```

**Comment by DTHNhung** [comment]
ready

**Pull request merged**